subprojects/docs/src/samples/play/advanced/build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 model {
     components {
         play {
-            platform play: '2.3.6', scala: '2.10'
+            platform play: '2.5.18', scala: '2.11'
```

Comment:
After this change, I got the following error on CI:

```
/home/tcagent1/agent/work/6031ced305d8bea4/subprojects/platform-play/build/tmp/test files/AbstractIntegrationSpec/unknown-test/94de2/play/advanced/build/src/play/binary/twirlTemplatesScalaSources/views/html/fields.template.scala:29: Any does not take parameters
    <label for=""""),_display_(/*4.18*/elements/*4.26*/.id),format.raw/*4.29*/("""">"""),_display_(/*4.32*/elements/*4.40*/.label(elements.lang)),format.raw/*4.61*/("""</label>
                                                                                                                                 ^
/home/tcagent1/agent/work/6031ced305d8bea4/subprojects/platform-play/build/tmp/test files/AbstractIntegrationSpec/unknown-test/94de2/play/advanced/build/src/play/binary/twirlTemplatesScalaSources/views/html/fields.template.scala:30: value lang is not a member of views.html.helper.FieldElements
    <span class="text-error">"""),_display_(/*5.31*/elements/*5.39*/.errors(elements.lang).mkString(", ")),format.raw/*5.76*/("""</span>
                                                                                     ^
/home/tcagent1/agent/work/6031ced305d8bea4/subprojects/platform-play/build/tmp/test files/AbstractIntegrationSpec/unknown-test/94de2/play/advanced/build/src/play/binary/twirlTemplatesScalaSources/views/html/person.template.scala:45: could not find implicit value for parameter messages: play.api.i18n.Messages
        """),_display_(/*20.10*/helper/*20.16*/.inputText(personForm("name"), '_label -> "What is your name?", '_showConstraints -> false)),format.raw/*20.107*/("""
                                                         ^
/home/tcagent1/agent/work/6031ced305d8bea4/subprojects/platform-play/build/tmp/test files/AbstractIntegrationSpec/unknown-test/94de2/play/advanced/build/src/play/binary/twirlTemplatesScalaSources/views/html/person.template.scala:46: could not find implicit value for parameter messages: play.api.i18n.Messages
        """),_display_(/*21.10*/helper/*21.16*/.inputText(personForm("quest"), '_label -> "What is your quest?", '_showConstraints -> false)),format.raw/*21.109*/("""
                                                         ^
/home/tcagent1/agent/work/6031ced305d8bea4/subprojects/platform-play/build/tmp/test files/AbstractIntegrationSpec/unknown-test/94de2/play/advanced/build/src/play/binary/twirlTemplatesScalaSources/views/html/person.template.scala:47: could not find implicit value for parameter messages: play.api.i18n.Messages
        """),_display_(/*22.10*/helper/*22.16*/.inputText(personForm("favoriteColor"), '_label -> "What is your favorite color?")),format.raw/*22.98*/("""
                                                         ^
5 errors found

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compilePlayBinaryScala'.
> Compilation failed
```

Could you tell me why this happens? I'd appreciate it because I'm not an expert as you.

Reply:
Sorry about that. The errors are related to compatibility problems between Play 2.3 and 2.5. I've fixed them now.

Out of curiosity, where can I see the build status?

Reply:
Thank you very much. We have an internal CI to run all builds. For you, you could run `.gradlew platformPlay:check` to run all tests under `platformPlay` subprojects locally.

Reply:
Looks still incorrect:

```
C:\tcagent1\work\6031ced305d8bea4\subprojects\platform-play\build\tmp\test files\AbstractIntegrationSpec\unknown-test\19jyh\play\advanced\build\src\play\binary\routesScalaSources\hello\Routes.scala:29: Symbol 'type java.util.concurrent.CompletionStage' is missing from the classpath.
This symbol is required by 'method play.core.routing.HandlerInvokerFactory.javaJsonPromiseWebSocket'.
Make sure that type CompletionStage is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
A full rebuild may help if 'HandlerInvokerFactory.class' was compiled against an incompatible version of java.util.concurrent.
  import ReverseRouteContext.empty
         ^
one error found

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compilePlayBinaryScala'.
> Compilation failed
```

Maybe you could run `./gradlew platformPlay:quickTest` to reproduce it?

Reply:
Should be good now. :-)